identity-service/src/remoteConfig.js

```diff
@@ -1,5 +1,27 @@
 const REMOTE_CONFIG_FEATURE = 'remote_config'
 
+const DISCOVERY_NOTIFICATION_MAPPING = 'discovery_notification_mapping'
+const MappingVariable = {
+  PushRepost: 'push_repost',
```

Comment:
can you use the MappingVariable constants for these values?

Reply:
what do you mean?